### PR TITLE
Update README.md according to API change (addon instead of mixin)

### PR DESCRIPTION
**Pull request by rxaviers**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,96 +1,117 @@
...
- - A string value representing the CLDR indicator for the desired locale used in formatting the amount. Most apps will pull this value from the component's state or the state of one of its parent compenents.
+
 - **currency** - required
- - A string value representing the currency type (USD, EUR, etc)
+ - A 3-letter string currency code as defined by ISO 4217 (e.g., USD, EUR, CNY, etc).
 - **value** - required
  - The numeric value to be formatted
 - **options**
- - An optional set of options to further format the value. See the [numberFormatter](https://github.com/jquery/globalize/blob/master/doc/api/number/number-formatter.md) docs in Globalize for more info on specific options
+ - An optional set of options to further format the value. See the [currencyFormatter][] docs in Globalize for more info on specific options
```
Comment (kborchers) [inline README.md]: missing link

File: README.md [hunk 2]
```diff
@@ -1,96 +1,117 @@
...
 - Accounting format with EUR in Portuguese
 
-  `<FormatCurrency locale="pt-BR" currency="EUR" value={-150} options={{ style: "accounting" }} />`
+  `<FormatCurrency currency="EUR" value={-150} options={{ style: "accounting" }} />`
+
+Result: `<span>(€150,00)</span>` when using the `pt` (Portuguese) locale.
+
+### FormatDate
+
+It allows to convert dates and times from their internal representations to textual form in a language-independent manner. Your code can conveniently control the length of the formatted date, time, datetime. See [dateFormatter][] docs in Globalize for more information.
```
Comment (kborchers) [inline README.md]: missing link

File: README.md (hunk 1 above)
Comment (kborchers) [inline README.md]: nvm, diff is confusing

File: README.md (hunk 2 above)
Comment (kborchers) [inline README.md]: nvm, diff is confusing

Comment (rxaviers) [inline README.md]: ok, ignoring...

File: README.md (hunk 1 above)
Comment (rxaviers) [inline README.md]: ok, ignoring...

**Pull request merged**